.github/workflows/release.yml

```diff
@@ -37,10 +37,6 @@ jobs:
         run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          distribution: goreleaser
-          version: v1.2.5
-          args: release --rm-dist
```

Comment:
:thinking:

Reply:
I moved the go releaser call to make so we don't duplicate the go releaser version on the two CI files + we can call the dry run easily locally. WDYT ?